apps/explorer/src/components/Activity/index.tsx

```diff
@@ -60,7 +60,12 @@ export function Activity({
             >
                 <div className="relative">
                     <TabList>
-                        <Tab>Transaction Blocks</Tab>
+                        <Tab>
+                            <div className="hidden md:block">
```

Comment:
I talked to Abhi and I think these breakpoints are wrong

Reply:
Should we go down in breakpoint to `sm` (640)? Tablet is `768px` so I just aim for right below it. @mystie711 Could you give advice here.

Reply:
We should just create a new breakpoint `ss` that maps to the 390px width